src/share/classes/com/sun/javatest/regtest/tool/Tool.java

```diff
@@ -642,6 +643,13 @@ public void process(String opt, String arg) {
             }
         },
 
+        new Option(NONE, MAIN, null, "-ve", "-verifyexclude") {
```

Comment:
I might have gone for `--verify-exclude` but this is OK

Reply:
The existing options are all "very old" and the style goes all the way back to the beginning of java-time, in the mid 90s. We have started using the double-dash form for recent options, especially where like-named options appear in other tools -- most notable `--class-path` etc.

When we started introducing the double-dash for in JDK 9 ([JEP 293](https://openjdk.org/jeps/293)) there was general agreement that we should not go back and "gratuitously" update existing old-form options. But equally, there was general agreement to use the new double-dash form for new options.